### PR TITLE
Ruleset Files moved to rules/decoders section

### PR DESCRIPTION
**Pull request by juankaromo**
Hi team,

This PR places files management within its appropriate section (rules or decoders), and change the new file creation place to it.

This PR solves #1298 

Regards

### CODE REVIEW

File: public/controllers/management/files.js [hunk 1]
```diff
@@ -86,6 +147,21 @@ export class FilesController {
     }
   }
 
+  addNewFile = type => {
```
Comment (jesusgn90) [inline public/controllers/management/files.js]: `addNewFile = type => {` --> `addNewFile(type) {`

File: public/controllers/management/files.js [hunk 2]
```diff
@@ -86,6 +147,21 @@ export class FilesController {
...
+    this.$scope.editingFile = true;
+    this.$scope.newFile = true;
+    this.$scope.newFileName = '';
+    this.$scope.selectedFileName = this.$scope.selectedRulesetTab;
+    this.$scope.selectedItem = { file: 'new file' };
+    this.$scope.fetchedXML = '<!-- Modify it at your will. -->';
+    this.$scope.type = type;
+    this.$scope.cancelSaveAndOverwrite();
+    if (!this.$scope.$$phase) $scope.$digest();
```
Comment (jesusgn90) [inline public/controllers/management/files.js]: `if (!this.$scope.$$phase) $scope.$digest();` --> `this.$scope.$applyAsync();`

File: public/controllers/management/files.js [hunk 3]
```diff
@@ -86,6 +147,21 @@ export class FilesController {
...
+    this.$location.search('editingFile', true);
+    this.appState.setNavigation({ status: true });
+    this.$scope.$emit('fetchedFile', { data: this.$scope.fetchedXML });
+  };
```
Comment (jesusgn90) [inline public/controllers/management/files.js]: Remove `;`

**Comment by juankaromo** [comment]
Other fixes this PR includes:

- [x] Re-fetch of data when changing dynamic columns
- [x] The current rules switch did not save the status when entering the detail of a rule.
- [x] Resizable columns did not work in tables with expandable rows (ng-repeat-start, ng-repeat-end).
- [x] Wrong colspan on some boards.

**Comment by jesusgn90** [comment]
Good job @juankaromo !

**Pull request merged**